aloe/fs.py

```diff
@@ -80,7 +80,7 @@ def find_feature_directories(cls, dir_):
         """
 
         # A set of package directories discovered
-        packages = set()
+        packages = set(dir_)
```

Comment:
Probably needs more explanation, the whole implementation is not that clear to me. Probably explain the desired results from the README here.